storage/innobase/xtrabackup/src/read_filt.cc

```diff
@@ -119,15 +121,15 @@ static void rf_bitmap_init(
  Get the next batch of pages for the bitmap read filter.  */
 static void rf_bitmap_get_next_batch(
     /*=====================*/
-    xb_read_filt_ctxt_t *ctxt,     /*!<in/out: read filter
-                                   context */
+    xb_fil_cur_t *cursor,          /*!< in/out: source file cursor */
     ib_uint64_t *read_batch_start, /*!<out: starting read
                                    offset in bytes for the
                                    next batch of pages */
     ib_uint64_t *read_batch_len)   /*!<out: length in
                                    bytes of the next batch
                                    of pages */
 {
+  xb_read_filt_ctxt_t *ctxt = &cursor->read_filter_ctxt;
```

Comment:
are we using cursor? looks like read_fil context is enough?

Reply:
It is common to all read filter need it reading file inside rf_page_tracking_get_next_batch.